app/users/resources.py

```diff
@@ -0,0 +1,48 @@
+from flask import request
+from flask_restful import Resource
+import json
+
+from app.users.models import User
+from app import db
+
+class UserListResource(Resource):
+    def get(self):
+        users = User.query.all()
+        return {
+            'result': [{'id': user.id, 'name': user.name} for user in users]
+            }
+
+    def post(self):
+        # import pdb; pdb.set_trace()
+        new_user_data = json.loads(request.form['data'])  # this is a dictionary
+        new_user = User(new_user_data.get('name'), new_user_data.get('email'), new_user_data.get('password'))
```

Comment:
So something we might want to add soonish is a validator. `request.form` might not have a `data` key (so a `KeyError` might happen), and if it does, it might not be a dictionary in string format (so a `ValueError` might happen). We would want to catch these scenarios and raise an appropriate error message. Also one or all of name, email, and password might not be provided in the dictionary, so we would also want to validate that all three are provided and raise an error if they aren't.